sphinx/source/docs/dev_guide/setup.rst

```diff
@@ -297,65 +498,112 @@ You should see output similar to:
+command(s):
+
+.. tabs::
+
+    .. code-tab:: sh Linux/macOS
+
+        BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
+
+    .. code-tab:: PowerShell Windows (PS)
+
+        $Env:BOKEH_RESOURCES = "inline"
+        python.exe .\examples\plotting\file\iris.py
+
+    .. code-tab:: doscon Windows (CMD)
 
-    BOKEH_RESOURCES=inline python examples/plotting/file/iris.py
+        set BOKEH_RESOURCES=inline
+        python examples\plotting\file\iris.py
 
-which will create a file ``iris.html`` locally and open up a web browser.
+This creates a file ``iris.html`` locally. When you open this file in a web
+browser, it should display this visualization:
 
 .. image:: /_images/bokeh_iris_html.png
     :scale: 50 %
     :align: center
 
-The variable ``BOKEH_RESOURCES`` determines where the css and JavaScript
-resources required by bokeh are found. By specifying ``inline`` we are using
-the version of BokehJS we just built to include the resources inline as part of
-the HTML file. The ``BOKEH_RESOURCES`` variable is required as the default
-behavior is to use CDN resources.
+Run Bokeh Server
+~~~~~~~~~~~~~~~~
 
-Another method of running bokeh is as a server. An example of this mode of
-operation can be run using the command:
+Another way to use Bokeh is as a :term:`server <Server>`. Set the
+:ref:`environment variable <devguide_setup_environment_variables>`
+``BOKEH_DEV=false`` and run the ``bokeh serve`` command in the *source
+checkout* directory:
 
-.. code-block:: sh
+.. tabs::
+
+    .. code-tab:: sh Linux/macOS
+
+        BOKEH_DEV=false python -m bokeh serve --show examples/app/sliders.py
+
+    .. code-tab:: PowerShell Windows (PS)
 
-    python -m bokeh serve --show examples/app/sliders.py
+        $Env:BOKEH_DEV = "False"
+        python.exe -m bokeh serve --show .\examples\app\sliders.py
 
-which will open up a browser with an interactive figure.
+    .. code-tab:: doscon Windows (CMD)
+
+        set BOKEH_DEV=false
+        python -m bokeh serve --show \examples\app\sliders.py
+
+This should open up a browser with an interactive figure:
 
 .. image:: /_images/bokeh_app_sliders.png
     :scale: 50 %
     :align: center
 
 All the sliders allow interactive control of the sine wave, with each update
-redrawing the line with the new parameters. The ``--show`` option opens the
-web browser to the appropriate address, the default is ``localhost:5006``.
-
-If you have any problems with the steps here, please `contact the developers`_.
+redrawing the line with the new parameters. The ``--show`` option opens a
+web browser, the default URL for the Bokeh server is ``localhost:5006``.
 
+.. note::
+    Updating an existing development environment does not always work as
+    expected. If you get errors after updating an older environment, use
+    ``conda remove --name bkdev --all``, delete your local ``bokeh`` folder,
+    and start afresh, following the steps in this guide from
+    :ref:`the beginning <devguide_setup_preliminaries>`.
```

Comment:
Should this note be up in the conda section? It seems odd for it to be in "Run Bokeh Server" Or maybe a dedicated troubleshooting section?